code/game/machinery/vending_vr.dm

```diff
@@ -182,64 +182,64 @@
-					/obj/item/clothing/gloves/fluff/siren = 200,
-					/obj/item/clothing/gloves/white = 200,
-					/obj/item/clothing/gloves/duty = 200,
-					/obj/item/clothing/shoes/athletic = 100,
-					/obj/item/clothing/shoes/boots/fluff/siren = 100,
-					/obj/item/clothing/shoes/slippers = 100,
-					/obj/item/clothing/shoes/boots/cowboy/classic = 100,
-					/obj/item/clothing/shoes/boots/cowboy = 100,
-					/obj/item/clothing/shoes/boots/duty = 200,
-					/obj/item/clothing/shoes/flats/white/color = 100,
-					/obj/item/clothing/shoes/flipflop = 100,
-					/obj/item/clothing/shoes/heels = 100,
-					/obj/item/clothing/shoes/hitops/black = 100,
-					/obj/item/clothing/shoes/hitops/blue = 100,
-					/obj/item/clothing/shoes/hitops/green = 100,
-					/obj/item/clothing/shoes/hitops/orange = 100,
-					/obj/item/clothing/shoes/hitops/purple = 100,
-					/obj/item/clothing/shoes/hitops/red = 100,
-					/obj/item/clothing/shoes/flats/white/color = 100,
-					/obj/item/clothing/shoes/hitops/yellow = 100,
-					/obj/item/clothing/shoes/boots/jackboots = 100,
-					/obj/item/clothing/shoes/boots/jungle = 200,
-					/obj/item/clothing/shoes/black/cuffs = 100,
-					/obj/item/clothing/shoes/black/cuffs/blue = 100,
-					/obj/item/clothing/shoes/black/cuffs/red = 100,
-					/obj/item/clothing/shoes/sandal = 100,
-					/obj/item/clothing/shoes/black = 100,
-					/obj/item/clothing/shoes/blue = 100,
-					/obj/item/clothing/shoes/brown = 100,
-					/obj/item/clothing/shoes/laceup = 100,
-					/obj/item/clothing/shoes/green = 100,
-					/obj/item/clothing/shoes/leather = 100,
-					/obj/item/clothing/shoes/orange = 100,
-					/obj/item/clothing/shoes/purple = 100,
-					/obj/item/clothing/shoes/red = 100,
-					/obj/item/clothing/shoes/white = 100,
-					/obj/item/clothing/shoes/yellow = 100,
-					/obj/item/clothing/shoes/skater = 100,
-					/obj/item/clothing/shoes/boots/cowboy/snakeskin = 100,
-					/obj/item/clothing/shoes/boots/jackboots/toeless = 100,
-					/obj/item/clothing/shoes/boots/workboots/toeless = 100,
-					/obj/item/clothing/shoes/boots/winter = 100,
-					/obj/item/clothing/shoes/boots/workboots = 100,
-					/obj/item/clothing/shoes/footwraps = 100)
+	prices = list(/obj/item/clothing/gloves/evening = 50,
+					/obj/item/clothing/gloves/fingerless = 50,
+					/obj/item/clothing/gloves/black = 50,
+					/obj/item/clothing/gloves/blue = 50,
+					/obj/item/clothing/gloves/brown = 50,
+					/obj/item/clothing/gloves/color = 50,
+					/obj/item/clothing/gloves/green = 50,
+					/obj/item/clothing/gloves/grey = 50,
+					/obj/item/clothing/gloves/sterile/latex = 50,
+					/obj/item/clothing/gloves/light_brown = 50,
+					/obj/item/clothing/gloves/sterile/nitrile = 50,
+					/obj/item/clothing/gloves/orange = 50,
+					/obj/item/clothing/gloves/purple = 50,
+					/obj/item/clothing/gloves/red = 50,
+					/obj/item/clothing/gloves/fluff/siren = 50,
+					/obj/item/clothing/gloves/white = 50,
+					/obj/item/clothing/gloves/duty = 50,
```

Comment:
Gloves should be 100 each

Reply:
Gloves PR has been merged at polaris
https://github.com/PolarisSS13/Polaris/pull/7287